components/Elevation/src/MDCElevatable.h

```diff
@@ -31,11 +31,12 @@
 
  Use this block to respond to elevation changes in the view or its ancestor views.
 
- @param elevation The @c mdc_currentElevation plus the @c mdc_currentElevation of all ancestor
- views.
+ @param absoluteElevation The @c mdc_currentElevation plus the @c mdc_currentElevation of all
```

Comment:
Nit: "The @c mdc_currentElevation plus the @c mdc_baseElevation of the object. This equates to the @c mdc_absoluteElevation in @c MaterialElevationResponding."

Reply:
mdc_baseElevation isn't a property in this protocol, i tried to explain it using what this protocol provides to make it easier on the user. Do you still want me to update to your copy?

Reply:
talked offline, resolving.